go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/tektoncd/pipeline
 
-go 1.22
+go 1.22.3
```

Comment:
NIT: I think we typically avoid changing the Go version in the same PR as introducing a new feature.